### PR TITLE
React refactor

### PR DESCRIPTION
**Pull request by JohnForster**

### CODE REVIEW

File: buildScripts/server.js [hunk 1]
```diff
@@ -22,3 +22,9 @@ app.use(require('webpack-dev-middleware')(compiler, {
   noInfo: true,
   publicPath: config.output.publicPath
 }));
+
```
Comment (archmagos) [inline buildScripts/server.js]: Do we need these? Rails backend seems to resolve the cross origin issues

File: src/tictactoe.js [hunk 2]
```diff
@@ -0,0 +1,112 @@
+function Square(props) {
```
Comment (archmagos) [inline src/tictactoe.js:1]: Do we need this file at all?

**Review by archmagos — CHANGES_REQUESTED** [review]
Thanks John - see comments in review

**Comment by YvCodeHong** [comment]
I don't have a problem with this.

**Review by YvCodeHong — DISMISSED** [review]

**Review by archmagos — APPROVED** [review]

File: src/tictactoe.js (hunk 2 above)
Comment (JohnForster) [reply src/tictactoe.js:1]: Keep for now, as it is a useful reference?

**Review by YvCodeHong — APPROVED** [review]

**Pull request merged**